docs/cli/odigos_diagnose.mdx

```diff
@@ -3,7 +3,7 @@ title: "odigos diagnose"
 sidebarTitle: "odigos diagnose"
 ---
 
-Retrieves Logs of all Odigos Components in the odigos-system namespace and CRDs of Actions, Instrumentation resources and more.<br />
+Retrieves Logs of all Odigos components in the odigos-system namespace and CRDs of actions, instrumentation resources and more.<br />
```

Comment:
maybe Actions should be capitalized since it's the name of an actual object/feature?